netbox/extras/forms/model_forms.py

```diff
@@ -129,10 +130,10 @@ class Meta:
 
     def __init__(self, *args, initial=None, **kwargs):
 
-        # Convert any parameters delivered via initial data to JSON data
+        # Convert any parameters delivered via initial data to dict
```

Comment:
This needs to accept JSON data in order for cloning to work. Rather than changing the form back to taking a QueryDict, we should change the originating link to send JSON.